File: Pemro1_modul5/Bahasa_C/PRAK502-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
    int hasil = nilai1 - nilai2;
    if (hasil < 0){
        hasil = -hasil;
    }
    return hasil;
}

int mutlak(int angka){
    if (angka < 0){
        angka = -angka;
    }
    return angka;
}

int main()
{
    int a, b, c, d;
    int hasil;

    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);

    hasil = hitung(a,b) + hitung(c,d);  
    printf("%d", mutlak(hasil));        
    return 0;
}

File: Pemro1_modul5/Bahasa_Python/PRAK502-2510817320010-PutriDeissytaHerliyana.py
def hitung(nilai1, nilai2):
    hasil = nilai1 - nilai2
    if hasil < 0:
        hasil = -hasil
    return hasil

def mutlak(angka):
    if angka < 0:
        angka = -angka
    return angka

a, c, b, d = map(int, input().split())

hasil = hitung(a, b) + hitung(c, d)
print(mutlak(hasil))
